prep.py

```diff
@@ -0,0 +1,40 @@
+"""prep.py
+
+Cleans and prepares the input data to be used by the model to train or predict
+the Titanic survivors.
+
+Input Data and relevant transformations:
+`pclass` is the ticket class (1st, 2nd, or 3rd), which should be transformed
+into two dummy variables.
+`sex` should be transformed into a dummy variable
+`age` should be normalized
+`sibsp` should be normalized
+`parch` should be normalized
+`ticket` is an ID variable that is not relevant for the model
+`fare` should be normalized
+`cabin` [[[potentially relevant, but not sure how to deal with this just yet]]]
+`embarked` [[[not sure if relevant]]]
```

Comment:
We need to figure out if `cabin` and `embarked` should be used for the model, and how.